### PR TITLE
fix trackovania

### PR DESCRIPTION
**Pull request by Kamahl19**

### CODE REVIEW

File: root/socialnet/install_sn.php [hunk 1]
```diff
@@ -30,7 +30,7 @@
 }
 
 // The name of the mod to be displayed during installation.
-$mod_name = 'Social Network';
+$mod_name = 'phpBB Social Network';
```
Comment (senky) [inline root/socialnet/install_sn.php:33]: this was already merged, remove it, please, and do `pull` to match latest changes.

File: root/socialnet/update_sn_0.7.0.php [hunk 2]
```diff
@@ -981,8 +981,6 @@ function phpbb_SN_umil_send($action, $version)
 	);
 	$query = http_build_query(array('q' => base64_encode(serialize($data))));
 
-	//$query = http_build_query($data);
```
Comment (senky) [inline root/socialnet/update_sn_0.7.0.php:984]: What was this? Old method that sent all $data parameters separatelly, and from now it serialises and sends only in one parameter `q`?

File: root/socialnet/update_sn_0.7.0.php [hunk 3]
```diff
@@ -30,7 +30,7 @@
 }
 
 // The name of the mod to be displayed during installation.
-$mod_name = 'Social Network - update';
+$mod_name = 'phpBB Social Network - update';
```
Comment (senky) [inline root/socialnet/update_sn_0.7.0.php:33]: again, `pull` before.

Comment (Kamahl19) [inline root/socialnet/update_sn_0.7.0.php:33]: thats strange because i have pulled and my repo was up to date before creating a new branch and editing it

Comment (Kamahl19) [inline root/socialnet/update_sn_0.7.0.php:33]: so what should i do now

Comment (senky) [inline root/socialnet/update_sn_0.7.0.php:33]: https://github.com/phpBB-Social-Network/phpBB-Social-Network/commit/b691fdf2dd6d221c55278d72219e6184a6a05723 was merged already. This is the reason why this PR cannot be merged automatically. Try simply `git pull upstream develop` on your track_fix branch. Then push changes and we will see.

Comment (Kamahl19) [inline root/socialnet/update_sn_0.7.0.php:33]: Martin@MARTIN-PC /D/workspace/phpBB-Social-Network (track_fix)
$ git pull upstream develop
From git://github.com/Kamahl19/phpBB-Social-Network
- branch            develop    -> FETCH_HEAD
  Already up-to-date.

Martin@MARTIN-PC /D/workspace/phpBB-Social-Network (track_fix)
$ git push origin track_fix
Everything up-to-date

nothing has happend. once again, why does this happen? i dont understand

**Comment by Culprit-CZ** [comment]
Make neccessary changes to be mergable, otherwise the function is my, you both should make merge approval

**Comment by senky** [comment]
Ok, we will see how it merges.

MA

**Comment by Kamahl19** [comment]
so how can we merge it?

**Comment by senky** [comment]
I will merge it.

**Pull request merged**